test_programs/execution_success/as_witness/src/main.nr

```diff
@@ -0,0 +1,5 @@
+// Simple example of checking where two arrays are different
+fn main(a: Field) -> pub Field {
+    std::as_witness(a);
+    a
+}
```

Comment:
Does this still fail even though it is not unconstrained? Runs fine for me on master.

Or are we relying on the `brillig_*` version of all the tests?

Reply:
Yeah, this will be picked up by the `brillig_` version of the test. Can verify by using `nargo compile --force-brillig` with the nightly version.